src/core/server/graph/schema/schema.graphql

```diff
@@ -4579,6 +4603,16 @@ input StaffConfigurationInput {
   label is the string used when displaying the STAFF badge to users.
   """
   label: String
+
```

Comment:
I'd suggest the introduction of a new `staffLabel`. Mark the existing `label` as `@deprecated` and replace usage with `staffLabel`.

Reply:
would this require a migration?

Reply:
Nope! We just adjust it on the resolver level, not in the database.